app/helpers/provider/admin/dashboards_helper.rb

```diff
@@ -37,7 +37,12 @@ def dashboard_navigation_link(link_text, path, options = {})
   end
 
   def dashboard_collection_link(singular_name, collection, path, options = {})
-    link_text = pluralize(number_to_human(collection.size), singular_name, options.fetch(:plural, nil))
+    limit = options.fetch(:limit, nil)
+    collection_size = collection.size
+    collection_size = limit if limit && collection_size > limit
+    human_size = number_to_human(collection_size)
+    human_size += "+" if collection_size == limit
+    link_text = pluralize(human_size, singular_name, options.fetch(:plural, nil))
```

Comment:
Previous functionality remains intact, but now it accepts the `:limit` option to show the `+` instead of the actual size.